docs/_renderer.py

```diff
@@ -279,7 +258,7 @@ def read_file(file: str | Path, root_dir: str | Path | None = None) -> FileConte
 
 
 def check_if_missing_expected_example(el, converted):
-    if re.search(r"(^|\n)#{2,6} Examples\n", converted):
+    if re.search(r"(^|\n)#{2,6} Examples", converted):
```

Comment:
quartodoc adds class names to the sections now, so we had to remove the newline from the regex.